src/components/Table/Table.tsx

```diff
@@ -28,15 +45,35 @@ interface TableProps {
    * Set to true if data is loading.
    */
   isLoading?: boolean;
-  /**
-   * Responsive tables allow tables to be scrolled horizontally with ease.
-   * If table overruns horizontal width of container, allow for horizontal scrolling.
-   */
-  isResponsive?: boolean;
   /**
    * Adds zebra-striping to any table row within the table body.
    */
   isStriped?: boolean;
+  /**
+   * Callback function to fire on sorting one of the table headers.
+   */
+  onSort?: (event: EventWithColumnKey) => void;
+  /**
+   * Set the maximum width and height and enable scrolling within the container when the table grows
+   * past those values. Useful for when we want to render a large table but not force the parent container
+   * to grow and instead make the user scroll.
+   */
+  scroll?: {
+    x: number | undefined;
+    y: number | undefined;
+  };
```

Comment:
It will be easy enough to add a unit to this later that would accept either 'px' or '%'. Can totally make it backwards compatible buy just saying: "if no unit is supplied, use pixels".